app/src/Tasks/index.js

```diff
@@ -165,6 +161,17 @@ const QuoteList = ({ quote: { quotation, author } }) => (
   </span>
 );
 
+// const PairsList = ({ pairs }) => (
+//   <ul>
+//     {pairs.map(({ id, title, icon }) => (
```

Comment:
you're expecting `pairs` to be an array here. but you set it as `{}` in the initial value in setState.

Did you want an array or object data type?